client/src/partials/vouchers/simple.js

```diff
@@ -59,19 +76,58 @@ function SimpleJournalVoucherController(AppCache, Vouchers, Accounts, Session, u
       return;
     }
 
+    // description prefix
+    vm.voucher.description =  String(vm.descriptionPrefix).concat('/', vm.voucher.description);
+
     // submit the voucher
     return Vouchers.createSimple(vm.voucher)
     .then(function (res) {
-      Notify.success('FORM.INFO.UPDATE_SUCCESS');
 
-      /* setup the voucher object to init state */
+      // Generate the document
+      return Vouchers.report(res.uuid, 'pdf');
```

Comment:
Cool!

@sfount @mbayopanda we should sit down as a team and discuss how we will do reports/receipts.  It seems like we all have our own (good) ideas.  We should formalize those before going in too many different directions.